exams/pearson/writers_test.py

```diff
@@ -186,7 +187,7 @@ def test_write_skips_invalid_rows(self):  # pylint: disable=no-self-use
 
 
 @ddt.ddt
-class CDDWriterTest(TSVWriterTestCase):
+class CDDWriterTest(TSVWriterTestCase, TestCase):
```

Comment:
Please do not use `django.test.TestCase`: use `search.base.MockedESTestCase` or `ESTestCase`: `django.test.TestCase`could have some side effects now that we mock all the indexing functions.